plone/outputfilters/filters/resolveuid_and_caption.py

```diff
@@ -333,18 +334,22 @@ def unknown_starttag(self, tag, attrs):
             if (tag == 'a' or tag == 'area') and 'href' in attributes:
                 href = attributes['href']
                 scheme = urlsplit(href)[0]
```

Comment:
``scheme`` is redefined two lines below, so I think this line can be removed.

Reply:
Fixed.